### PR TITLE
chore: Make dark mode even darker

### PR DESCRIPTION
**Pull request by richardwerkman**
As a true dark mode fanboy the current home page still hurted my eyes.

That's why I turned:

image

into:

image

Todo:
- [ ] Fix strykerman to contain info support logo and kill the mutants text
- [x] Fix size of strykerman (pushes other content out of view without scrolling now)
- [x] Look into why the generated root is suddenly committed

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -71,10 +71,7 @@ function Home() {
               </Link>
             </div>
           </div>
-          <picture className={styles.heroImage}>
-            <source media="(min-width:662px)" srcSet="images/strykerman.png"></source>
-            <img src="images/gun.png" alt="Strykerman"></img>
-          </picture>
+          <img src="images/StrykerMan.svg" alt="Strykerman" className="stryker-man"></img>
```
Comment (hugo-vrijswijk) [inline src/pages/index.js:74]: You can probably still use [picture](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) for svg's as well if the size is set properly

File: src/css/custom.css [hunk 2]
```diff
@@ -55,3 +64,17 @@ html[data-theme='dark'] .header-github-link:before {
...
+  background-color: var(--ifm-color-primary-darker);
+  border-color: var(--ifm-color-primary-darkest);
+}
+
+:root[data-theme='dark'] .button.button--secondary.button--outline:not(.button--active):hover {
+  color: #ddd;
+  background-color: var(--ifm-color-primary-darkest);
+}
+
+.stryker-man {
```
Comment (hugo-vrijswijk) [inline src/css/custom.css:78]: This can be in the styling for the page

File: src/pages/index.js (hunk 1 above)
Comment (richardwerkman) [reply src/pages/index.js:74]: Well it did not show the image... Only by removing the picture tag I got it to show. I'll try something else with the picture tag

**Comment by hugo-vrijswijk** [comment]
@richardwerkman What is the status of this PR?

**Comment by richardwerkman** [comment]
The image should be updated to contain the infosupport logo and stryker name again. If we merge this they will be gone... Other than that it was finnished

**Review by hugo-vrijswijk — APPROVED** [review]

**Comment by hugo-vrijswijk** [comment]
Looks really nice!!

**Comment by nicojs** [comment]
Great job guys! I 😍 love this dark(er) mode 🌑

**Pull request merged**